tests/lax_numpy_test.py

```diff
@@ -2788,15 +2788,16 @@ def testSearchsortedNans(self, dtype, side, method):
     xshape=[(20,), (5, 4)],
     binshape=[(0,), (1,), (5,)],
     right=[True, False],
+    method=['scan', 'scan_unrolled', 'sort', 'compare_all'],
     reverse=[True, False],
     dtype=default_dtypes,
   )
-  def testDigitize(self, xshape, binshape, right, reverse, dtype):
+  def testDigitize(self, xshape, binshape, right, method, reverse, dtype):
     order = jnp.index_exp[::-1] if reverse else jnp.index_exp[:]
     rng = jtu.rand_default(self.rng())
     args_maker = lambda: [rng(xshape, dtype), jnp.sort(rng(binshape, dtype))[order]]
     np_fun = lambda x, bins: np.digitize(x, bins, right=right).astype('int32')
-    jnp_fun = lambda x, bins: jnp.digitize(x, bins, right=right)
+    jnp_fun = lambda x, bins: jnp.digitize(x, bins, right=right, method=method)
```

Comment:
This leaves untested calling the API without a method argument, which is the majority of uses.

How about we leave this test unchanged, and add another test that just parameterizes over passing `method`?